feature-libs/checkout/b2b/components/checkout-shipping-address/checkout-shipping-address.component.html

```diff
@@ -71,7 +71,6 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
         <cx-address-form
           [showTitleCode]="true"
           [setAsDefaultField]="!isGuestCheckout"
-          [addressData]="selectedAddress"
```

Comment:
redundant as we do not edit in checkout steps, therefore not populating the input of the address form. However, in the my-account/address-book, we can edit, therefore addressData is used to populate the input with existing address data